salsa_dancing_molecules/startup/use_custom_materials.py

```diff
@@ -0,0 +1,28 @@
+"""Module for handling custom materials."""
+from pickle import dump
+import os
+import importlib
+import sys
+
+
+def materials_to_pickles(python_modules_path, work_path, material_names):
+    """Import atoms object from python file and save as pickle.

```

Comment:
Please provide an example of what a custom material looks like. Or a reference to where an example can be found.